packages/astro/src/actions/runtime/middleware.ts

```diff
@@ -42,11 +42,13 @@ export const onRequest = defineMiddleware(async (context, next) => {
 });
 
 function nextWithLocalsStub(next: MiddlewareNext, locals: Locals) {
-	Object.defineProperty(locals, '_actionsInternal', {
-		writable: false,
-		value: {
-			getActionResult: () => undefined,
-		},
-	});
+	if (!locals._actionsInternal) {
```

Comment:
This covers the stub, but do we need a conditional check for [the assignment from middleware](https://github.com/withastro/astro/blob/main/packages/astro/src/actions/runtime/middleware.ts#L42) as well?

Reply:
yes probably